### DISCUSSION TITLE
Searching users is not working

### DISCUSSION THREAD

**Discussion by TrixDiaz** [munafio/chatify#272] [General]
I change my "name" field to firstName and LastName but while searching to other users I didn't get any results.

![image](https://user-images.githubusercontent.com/106210675/229951485-6e34971c-abcf-4073-90c8-ef628f4a63f3.png)

![image](https://user-images.githubusercontent.com/106210675/229951569-57f30fac-b3fa-4e52-91a5-a7878396f8d5.png)

**Comment by munafio** [comment]
@TrixDiaz you have to modify the search method at `messagesController.php` to meet your needs/changes.
